### PR TITLE
Refactor glimr api

### PR DESCRIPTION
**Pull request by tatyree**
Simplify the glimr api by changing out RestClient for HTTParty and by removing some of the client-interface-consistent response objects.

### CODE REVIEW

File: app/api_clients/glimr/requests/case_fees.rb [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
-      end
-
       def endpoint
         '/requestpayablecasefees'
       end
 
-      def body
+      def request_body
         {
           jurisdictionId: 8, # TODO: Remove when no longer required in API
```
Comment (digitalronin) [inline app/api_clients/glimr/requests/case_fees.rb]: Can we make this '8' a constant, with a meaningful name?

**Comment by digitalronin** [comment]
I'm liking this. Definitely happy to apply the same pattern to the govpay code.

File: app/api_clients/glimr/requests/case_fees.rb (hunk 1 above)
Comment (tatyree) [inline app/api_clients/glimr/requests/case_fees.rb]: As per our conversation, I'd rather leave it if/until we extract it into a gem.

**Comment by digitalronin** [comment]
LGTM. I think the encapsulation, rather than inheritance, is cleaner, and the code is easier to understand.

**Pull request merged**